File: packages/jsii-rosetta/test/translations/statements/statements_and_newlines.ts
function doThing() {
  const x = 1; // x seems to be equal to 1
  return x + 1;
}

function doThing2(x: number) {
  if (x == 1) {
    return true;
  }
  return false;
}

function doThing3() {
  const x = 1;
  return x + 1;
}

function doThing4() {
  let x = 1;
  x = 85;
}

File: packages/jsii-rosetta/test/translations/statements/statements_and_newlines.go
func doThing() *f64 {
	x := 1 // x seems to be equal to 1
	return jsii.Number(x + 1)
}

func doThing2(x *f64) *bool {
	if *x == 1 {
		return jsii.Boolean(true)
	}
	return jsii.Boolean(false)
}

func doThing3() *f64 {
	x := 1
	return jsii.Number(x + 1)
}

func doThing4() {
	x := 1
	x = 85
}
